### PR TITLE
VideoPress Onboarding V2: Intro View

### PR DESCRIPTION
**Pull request by roundhill**
#### Proposed Changes

* Adds styles and a few tweaks to some props to get "powered by VideoPress" displaying at the bottom of the intro view.

Screen Shot 2022-09-14 at 2 37 52 PM
Screen Shot 2022-09-14 at 2 38 21 PM

#### Testing Instructions

* Load up `setup/intro?flow=videopress`.
* Verify styles match the (i2) design.
* Try different screen sizes.
* Note: the button doesn't take you to the next step but I think we haven't set that up yet?
* Try a different flow like `setup/intro?flow=newsletter` and verify it still has `Jetpack powered` in the footer area.

#### Pre-merge Checklist

- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?

Related to pxWta-1z5-p2

### CODE REVIEW

**Review by jgcaruso — APPROVED** [review]
Code looks good and site matches designs!

Only note, "powered by" is missing in landscape mode for mobile devices but it seems to be like that for "link in bio" as well.

Screen Shot 2022-09-12 at 9 18 10 AM

**Comment by jgcaruso** [comment]
I think this is a duplication of work (or at least partially) of what is being done in https://github.com/Automattic/wp-calypso/pull/67736

Might want to confirm which one should stay or not before merging either one.

**Comment by roundhill** [comment]
> Might want to confirm which one should stay or not before merging either one.

We got this sorted out.

@eduardozulian is going to push a few CSS fixes to this by tomorrow.

File: client/landing/stepper/declarative-flow/internals/steps-repository/patterns/styles.scss [hunk 1]
```diff
@@ -35,7 +35,7 @@
 			}
 		}
 
-		.step-container__jetpack-powered {
+		.step-container__powered-by {
```
Comment (jgcaruso) [inline client/landing/stepper/declarative-flow/internals/steps-repository/patterns/styles.scss]: Is this change still needed now that a separate control is being used?

File: client/landing/stepper/declarative-flow/internals/steps-repository/intro/styles.scss [hunk 2]
```diff
@@ -85,7 +118,7 @@
 			margin-bottom: 0;
 		}
 	
-		.step-container__jetpack-powered {
+		.step-container__powered-by {
```
Comment (jgcaruso) [inline client/landing/stepper/declarative-flow/internals/steps-repository/intro/styles.scss]: Looks like another left over css change from before the merge?

**Review by jgcaruso — CHANGES_REQUESTED** [review]

**Comment by evilluendas** [comment]
I've just pushed a couple of very small tweaks to the position of the triangles on mobile, and made the progress bar background (in just this step) dark so it doesn't stand out so much.

I've noticed that this branch's Newsletter flow is slightly different to the one in trunk, but I imagine it's that they've already merged updates to trunk.

I'd like to update the poster for the video so it uses the first frame of the clip, but we can change that directly, right?

**Comment by roundhill** [comment]
> I'd like to update the poster for the video so it uses the first frame of the clip, but we can change that directly, right?

Yes, just DM me the image!

**Review by jgcaruso — CHANGES_REQUESTED** [review]
Looks good! Video plays, page looks nice in different screen sizes.

When testing the Jetpack flow though it looks like the font colour for the main text and "Jetpack Powered" was changed from black to white (dev on left, prod on right -- ignore the font alignment difference, the windows were just different heights)

Screen Shot 2022-09-21 at 10 52 32 AM

Not a blocker but the video also seems to take a couple seconds to start. That is probably something we'll need to optimize in the player if its possible (or maybe its like that for me because of sandboxing).

**Comment by roundhill** [comment]
> When testing the Jetpack flow though it looks like the font colour for the main text and "Jetpack Powered" was changed from black to white (dev on left, prod on right -- ignore the font alignment difference, the windows were just different heights)

Fixed via 8e52744!

> Not a blocker but the video also seems to take a couple seconds to start. That is probably something we'll need to optimize in the player if its possible (or maybe its like that for me because of sandboxing).

When we get the poster image added it might help?

**Comment by roundhill** [comment]
Bug? Our footer looks higher than the other flows:

Screen Shot 2022-09-21 at 4 00 52 PM

Or were you happy with that positioning @evilluendas?

**Comment by jgcaruso** [comment]
> When we get the poster image added it might help?

Hmm, do we show poster image for autoplay + no controls? It should also already have one by default, but something from the middle of the page which would be jarring if it flips between some random frame and the video start. Maybe we can set it to the first frame so it looks more natural... assuming it shows at all.

This is probably something where that "facade" diff you were working on would come in handy. Because if the black screen is because of the video info request we won't even know what the poster is yet to check.

**Review by jgcaruso — APPROVED** [review]
`newsletter` flow text colour is fixed, everything else looks good to me!

**Comment by roundhill** [comment]
> This is probably something where that "facade" diff you were working on would come in handy. Because if the black screen is because of the video info request we won't even know what the poster is yet to check.

Totally, maybe we should revisit the facade!

**Pull request merged**